fhirpipe/transform/fhir.py

```diff
@@ -175,11 +175,17 @@ def handle_array_attributes(attributes_in_array, row):
         assert length == 1 or len(val) == length, "mismatch in array lengths"
         length = len(val)
 
+    # Check if we have an array of literals
+    is_literal_array = len(attributes_in_array) == 1 and list(attributes_in_array.keys())[0] == ""
```

Comment:
can't literals be hndled by build_fhir_object ? that would make things a lot easier